### PR TITLE
[RAPPS] Improve Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to https://github.com/reactos/reactos/pull/6517

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

**Comment by GeoB99** [comment]
I cannot ship it due to merge conflicts, please fix them when you have time.

**Comment by AndreiMiloiu** [comment]
I fixed them.

File: base/applications/rapps/lang/ro-RO.rc [hunk 1]
```diff
@@ -19,88 +19,88 @@ BEGIN
...
     EDITTEXT IDC_DOWNLOAD_DIR_EDIT, 15, 86, 166, 12, WS_CHILD | WS_VISIBLE | WS_GROUP | ES_AUTOHSCROLL
-    PUSHBUTTON "Spe&cificare…", IDC_CHOOSE, 187, 85, 50, 14
-    AUTOCHECKBOX "Șt&erge aplicația de instalare după instalarea programului", IDC_DEL_AFTER_INSTALL, 16, 100, 218, 12
+    PUSHBUTTON "&Răsfoire", IDC_CHOOSE, 187, 85, 50, 14
+    AUTOCHECKBOX "Se șterge de instalare &după instalarea programului", IDC_DEL_AFTER_INSTALL, 16, 100, 218, 12
     GROUPBOX "Sursa programului", -1, 4, 118, 240, 46
-    AUTORADIOBUTTON "Folosește valorile implicite", IDC_SOURCE_DEFAULT, 15, 132, 121, 10, WS_GROUP | WS_TABSTOP
-    AUTORADIOBUTTON "Sursa specificată", IDC_USE_SOURCE, 15, 147, 74, 10
+    CONTROL "Se folosesc valorile implicite", IDC_SOURCE_DEFAULT, 15, 132, 121, 10, WS_GROUP | WS_TABSTOP
+    CONTROL "Sursa specificată", IDC_USE_SOURCE, 15, 147, 74, 10
```
Comment (GeoB99) [inline base/applications/rapps/lang/ro-RO.rc:80]: Nope! Using `CONTROL` without specifying a control style makes the resource compiler fail to build the resources:
```
/home/runner/work/reactos/reactos/src/base/applications/rapps/lang/ro-RO.rc:80: syntax error
```
Please revert all the specific changes back to `AUTORADIOBUTTON`s.

**Review by GeoB99 — DISMISSED** [review]

File: base/applications/rapps/lang/ro-RO.rc (hunk 1 above)
Comment (AndreiMiloiu) [reply base/applications/rapps/lang/ro-RO.rc]: Fixed!

**Pull request merged**